types/index.d.ts

```diff
@@ -19,6 +19,7 @@ export interface IJSONServerSpec {
 	username?: string;
 	password?: string;
 	description?: string;
+	pathPrefix?: string;
```

Comment:
I don't think this is correct.

`pathPrefix` is a property of the `webServer` object and is already defined on the `IWebServerSpec` interface above.